### ISSUE TITLE
css

### ISSUE THREAD

**Issue by naiduakash1122** [Chandukasireddy/stayrating#8]

**Comment by DoritosXL** [comment]
Hey what is this issue about, would you just like some css implementations?

**Comment by Chandukasireddy** [comment]
yes, some css design in the page. this page a repelecation of oyo website, but its a student service showing hostels.
